charts/k8ssandra/templates/stargate/service.yaml

```diff
@@ -17,12 +18,15 @@ spec:
     - name: rest
       port: 8082
       protocol: TCP
+    - name: health
+      port: 8084
+      protocol: TCP
     - name: metrics
       port: 8085
       protocol: TCP
-    - name: cql
+    - name: cassandra
       protocol: TCP
       port: 9042
   selector:
-    app: {{ .Release.Name }}-k8ssandra-stargate
+    app: {{ .Release.Name }}-{{ $datacenterName }}-stargate
```

Comment:
Similar to the name this will hopefully become something like this:

```yaml
selector:
    matchLabels:
      {{- include "k8ssandra-common.selectorLabels" . | nindent 6 }}
```

Reply:
https://helm.sh/docs/chart_template_guide/function_list/#nindent

It's relatively new to me. I am only familiar with a handful of functions. There are a lot 😄